### PR TITLE
Dashboard chart loader

### PR DESCRIPTION
**Pull request by Anchello**
Added the chart loader on Dashboard and updated another loaders

### CODE REVIEW

File: packages/manager/src/pages/dashboard/components/dashboard-portfolio-chart-section/dashboard-chart/dashboard-portfolio-chart-container.js [hunk 1]
```diff
@@ -58,6 +62,7 @@ const mapStateToProps = state => {
   const { assetChart, period } = state.dashboard;
   const { currency } = state.accountSettings;
   const { programs, funds } = state.dashboard;
+  console.log(state.dashboard);
```
Comment (pavelzubov) [inline packages/manager/src/pages/dashboard/components/dashboard-portfolio-chart-section/dashboard-chart/dashboard-portfolio-chart-container.js]: remove console.log

File: packages/shared/components/dashboard/dashboard-portfolio-events/dashboard-portfolio-event-loader/dashboard-portfolio-event-loader.js [hunk 2]
```diff
@@ -1,24 +1,20 @@
 import React from "react";
-import ContentLoader from "react-content-loader";
+import SvgLoader from "../../../svg-loader/svg-loader";
```
Comment (pavelzubov) [inline packages/shared/components/dashboard/dashboard-portfolio-events/dashboard-portfolio-event-loader/dashboard-portfolio-event-loader.js]: use way "shared/..."

File: packages/shared/components/table/components/table-loader-table-row.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
 import React from "react";
-import ContentLoader from "react-content-loader";
+import SvgLoader from "../../svg-loader/svg-loader";
```
Comment (pavelzubov) [inline packages/shared/components/table/components/table-loader-table-row.js]: use way "shared/..."

File: packages/shared/components/table/components/table-loader-table-row.js [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
 
 const TableLoaderTableRow = () => {
   return (
     <tr>
       <td className="table__cell programs-table__cell programs-table__cell--name">
-        <TableLoaderCell width={100} />
+        <TableLoaderCell width={100} id={23243} />
```
Comment (pavelzubov) [inline packages/shared/components/table/components/table-loader-table-row.js]: Why `id={23243}`?

**Comment by Anchello** [comment]
Fixed according to the comments, finished the loader for the dashboard page

**Pull request merged**